pandokia/top_level.html

```diff
@@ -16,12 +16,10 @@
 <p>
 <ul>
 
-<!--pretty sure we don't need this link, but I'm leaving it here for now -->
-<!--<li> <a href="CGINAME?query=day_report.2&test_run=jwst_latest">Latest JWST Report</a>-->
-
 <li> <a href="CGINAME?query=day_report.2&test_run=etc_hst_daily_latest"><b>Latest HST pyetc</b></a>
-<!-- <li> <a href="CGINAME?query=day_report.2&test_run=etc_jwst_daily_latest"><b>Latest Pandeia Report (Local)</b></a> -->
-<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_master_latest"><b>Latest Pandeia Report (AWS)</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_jwst_master_latest"><b>Latest JWST Pandeia Report</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_hst_master_latest"><b>Latest HST Pandeia Report</b></a>
+<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_roman_master_latest"><b>Latest Roman Pandeia Report</b></a>
```

Comment:
Love the forward thinking here.    It might be a hot minute before we have this running tho - does it make sense to leave this in but comment it out?

Reply:
Commented out roman related html tags